### ISSUE TITLE
Current release problem

### ISSUE THREAD

**Issue by matiux** [andres-montanez/Magallanes#203]
Hi,
I have a problem with release creation. In my remote server, the release dir is created, but current link, no. Infact, during deployment I get this error:
```
        Starting the Releasing
                Releasing on host 1.2.3.4 ... FAIL
        Finished the Releasing
```

I perform deploy of my Symfony2 project in this way:
```
 bin/mage deploy to:staging
```

I installed with composer, 1.0.4 version

Here is some configs.
environment file:
```
#staging
deployment:
  user: user
  from: ./
  to: /var/www/Project/Acme
  excludes:
    - .idea
    - composer.lock
    - spec
releases:
  enabled: true
  max: 5
  symlink: current
  directory: releases
hosts:
  - 1.2.3.4
tasks:
  pre-deploy:
  on-deploy:
  post-release:
  post-deploy:
```

And log which seems not contain any call to ln command needed to create "current" link

```
2015-03-08 00:45:09 -- Starting Magallanes
2015-03-08 00:45:09 -- Logging enabled
2015-03-08 00:45:09 -- Logging enabled: /var/www/Projects/Acme/.mage/logs/log-20150308-004509.log
2015-03-08 00:45:09 -- Deploy summary
2015-03-08 00:45:09 -- Environment: staging
2015-03-08 00:45:09 -- Release ID:  20150308004509
2015-03-08 00:45:09 -- 
2015-03-08 00:45:09 -- No Pre-Deployment tasks defined.
2015-03-08 00:45:09 -- Deploying to 1.2.3.4
2015-03-08 00:45:09 -- Running Deploy via TarGz (with Releases) [built-in] ... 
2015-03-08 00:45:09 -- Run remote command mkdir -p /var/www/Project/Acme/releases/20150308004509
2015-03-08 00:45:09 -- ---------------------------------
2015-03-08 00:45:09 -- ---- Executing: $ ssh  -p 22 -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no username@1.2.3.4 "sh -c \"mkdir -p /var/www/Project/Acme/releases/20150308004509\""
2015-03-08 00:45:15 -- 
2015-03-08 00:45:15 -- ---------------------------------
2015-03-08 00:45:15 -- ---------------------------------
2015-03-08 00:45:15 -- ---- Executing: $ tar cfzh /tmp/mageKKgmA9.tar.gz  --exclude=.git --exclude=.svn --exclude=.mage --exclude=.gitignore --exclude=.gitkeep --exclude=nohup.out --exclude=.idea --exclude=composer.lock --exclude=Acme.conf --exclude=spec --exclude=vendor -C ./ .
2015-03-08 00:45:16 -- 
2015-03-08 00:45:16 -- ---------------------------------
2015-03-08 00:45:16 -- ---------------------------------
2015-03-08 00:45:16 -- ---- Executing: $ scp  -P 22 /tmp/mageKKgmA9.tar.gz username@1.2.3.4:/var/www/Project/Acme/releases/20150308004509
2015-03-08 00:45:41 -- 
2015-03-08 00:45:41 -- ---------------------------------
2015-03-08 00:45:41 -- Run remote command cd /var/www/Project/Acme && cd releases/20150308004509 && tar xfz mageKKgmA9.tar.gz
2015-03-08 00:45:41 -- ---------------------------------
2015-03-08 00:45:41 -- ---- Executing: $ ssh  -p 22 -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no username@1.2.3.4 "sh -c \"cd /var/www/Project/Acme && cd releases/20150308004509 && tar xfz mageKKgmA9.tar.gz\""
2015-03-08 00:45:48 -- 
2015-03-08 00:45:48 -- ---------------------------------
2015-03-08 00:45:48 -- Run remote command cd /var/www/Project/Acme && cd releases/20150308004509 && rm -f mageKKgmA9.tar.gz
2015-03-08 00:45:48 -- ---------------------------------
2015-03-08 00:45:48 -- ---- Executing: $ ssh  -p 22 -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no username@1.2.3.4 "sh -c \"cd /var/www/Project/Acme && cd releases/20150308004509 && rm -f mageKKgmA9.tar.gz\""
2015-03-08 00:45:54 -- 
2015-03-08 00:45:54 -- ---------------------------------
2015-03-08 00:45:54 -- ---------------------------------
2015-03-08 00:45:54 -- ---- Executing: $ rm -f /tmp/mageKKgmA9 /tmp/mageKKgmA9.tar.gz
2015-03-08 00:45:54 -- 
2015-03-08 00:45:54 -- ---------------------------------
2015-03-08 00:45:54 -- OK
2015-03-08 00:45:54 -- Deployment to 1.2.3.4 completed: 1/1 tasks done.
2015-03-08 00:45:54 -- Starting the Releasing
2015-03-08 00:45:54 -- Releasing on host 1.2.3.4 ... 
2015-03-08 00:45:54 -- Run remote command cd /var/www/Project/Acme && ls -ld .
2015-03-08 00:45:54 -- ---------------------------------
2015-03-08 00:45:54 -- ---- Executing: $ ssh  -p 22 -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no username@1.2.3.4 "sh -c \"cd /var/www/Project/Acme && ls -ld .\""
2015-03-08 00:46:01 -- drwxr-xr-x 3 username username 4096 Mar  8 00:45 .
2015-03-08 00:46:01 -- ---------------------------------
2015-03-08 00:46:01 -- Run remote command cd /var/www/Project/Acme && chown -h username:username current && chown -R username:username releases/20150308004509 && chown username:username releases
2015-03-08 00:46:01 -- ---------------------------------
2015-03-08 00:46:01 -- ---- Executing: $ ssh  -p 22 -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no username@1.2.3.4 "sh -c \"cd /var/www/Project/Acme && chown -h username:username current && chown -R username:username releases/20150308004509 && chown username:username releases\""
2015-03-08 00:46:07 -- chown: cannot access ‘current’: No such file or directory
2015-03-08 00:46:07 -- ---------------------------------
2015-03-08 00:46:07 -- FAIL
2015-03-08 00:46:07 -- Finished the Releasing
2015-03-08 00:46:07 -- No Post-Deployment tasks defined.
2015-03-08 00:46:07 -- Time for deployment: 45 seconds.
2015-03-08 00:46:07 -- Average time per host: 45 seconds.
2015-03-08 00:46:07 -- Total time: 58 seconds.
2015-03-08 00:46:07 -- Finished Magallanes

```

**Comment by eps90** [comment]
Yes it's a bug in `master` version. Yesterday I fixed in `develop` so everything should be working now. Please try to install Magallanes via composer in `dev-develop` version.

**Comment by eps90** [comment]
Hey @matiux . This bug has been fixed in 1.0.5 version :)

**Comment by matiux** [comment]
Thanks. I will try tonight! ;-)
